Closure_Front_End/src/components/PieChart.vue

```diff
@@ -0,0 +1,22 @@
+
+<script>
+import { defineComponent } from 'vue';
+import { Pie } from 'vue3-chart-v2';
+
+export default defineComponent({
+  name: 'MonthlyChart',
+  extends: Pie,
+  mounted () {
+    // Overwriting base render method with actual data.
+    this.renderChart({
+      labels: ['Must', 'Choose from list', 'Choice'],
```

Comment:
How do we handle cases when one category (or more) require 0 points (so - isn't required)?

Reply:
in this graph it really doesn't matter as they just won't appear, but in the progress bars i'll probably need to drop the cornerStones bar when a certain track is taken (like cognition, which doesn't require corner stones)

Reply:
Cool, please do, and notice various courses may even have 0 `choice` or 0 `from_list` (do I hear a `0 must`?)